rpcs3/rpcs3qt/gui_settings.cpp

```diff
@@ -156,7 +156,7 @@ void gui_settings::SetCategoryVisibility(int cat, const bool& val)
 
 void gui_settings::ShowBox(bool confirm, const QString& title, const QString& text, const gui_save& entry, int* result = nullptr, QWidget* parent = nullptr, bool always_on_top = false)
 {
-	const std::string dialog_type = confirm ? "Confirmation" : "Info";
+	const std::string dialog_type = confirm ? (entry == gui::ib_confirm_fw ? "Error" : "Confirmation") : "Info";
```

Comment:
I'd prefer if "confirm" was refactored to directly use the enum QMessageBox::Icon instead of frankensteining in some special cases

Reply:
Fixed.